.github/workflows/trigger-benchmark.yml

```diff
@@ -0,0 +1,34 @@
+name: Check if needs trigger CircleCI benchmark
+
+on:
+  pull_request:
+   types: [opened, labeled, unlabeled, synchronize]
+
+jobs:
+  haslabel:
+    name: analyse labels
+    runs-on: ubuntu-latest
+    outputs:
+      benchmark: ${{ steps.haslabel.outputs.labeled-run-benchmark }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Labeled with run-benchmark
+        id: haslabel
+        uses: DanielTamkin/HasLabel@v1.0.4
+        with:
+          contains: 'run-benchmark'
+  perf-ci:
+    name: Trigger CI benchmarks
+    needs: haslabel
+    if: needs.haslabel.outputs.benchmark
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: curl-circle-ci
```

Comment:
https://github.com/marketplace/actions/trigger-circleci-pipeline

Reply:
@chayim with the following configuration ( as exemplified in ) this GH action is "forcing you" to use specific parameters and not the ones you want/need:

![image](https://user-images.githubusercontent.com/5832149/209541056-4472ee93-2ba9-4b54-876a-c85aaa615033.png)

IMHO I would avoid using the action before it maturing to the point in which you can use it without issues/properly setup the parameter names we want instead of those. 
TLDR: reverting back to the old curl

Reply:
WFM